nservicebus/azure-service-bus/multiple-namespaces-support.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Multiple namespace support
+summary: Configuring Azure ServiceBus transport to support different partitioning strategies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+reviewed: 2016-04-18
+---
+
+`Azure Service Bus` transport enables different _namespace partitioning strategies_ to cover different scenarios such as High Availability and multiple Data Center support or to overcome [service limits](https://azure.microsoft.com/en-us/documentation/articles/service-bus-quotas/).  
+
+_Namespace partitioning strategies_can be configured using Azure Service Bus configuration API, specifying namespace(s) to use.
+
+## Single namespace partitioning #
+
+The default _namespace partitioning strategy_ is `SingleNamespacePartitioning`. With this strategy, transport uses only a single namespace to send and receive messages.  
+Namespace connection string can be defined via code using `ConnectionString` extension:
```

Comment:
~~via code~~

you show in a snippet that it's done via code, so this is redundant

> extension

did you mean "extension method"?